api/envoy/data/tap/v3/common.proto

```diff
@@ -36,3 +38,14 @@ message Body {
   // <envoy_v3_api_field_config.tap.v3.OutputConfig.max_buffered_tx_bytes>` settings.
   bool truncated = 3;
 }
+
+// Connection properties.
+message Connection {
```

Comment:
just to double check, moving this doesn't break generated code compatibility? cc @envoyproxy/api-shepherds

Reply:
I think that should be ok, as the package name is the same.